run.js

```diff
@@ -103,13 +103,31 @@ const pkg = require('./package.json')
         state: 'open',
         labels: agendaLabel
       })
+
       console.log(`Fetching issues for ${r.owner}/${r.repo}: Found ${_agendaIssues.length}`)
       for (const i of _agendaIssues) {
         console.log(`Adding Issue: ${i.url}`)
         if (!agendaIssues.find((ii) => ii.url === i.url)) {
           agendaIssues.push(i)
         }
       }
+
+      const _agendaPr = await client.paginate('GET /repos/{owner}/{repo}/pulls', {
+        owner: r.owner,
+        repo: r.repo,
+        state: 'open',
+        labels: agendaLabel
+      })
```

Comment:
```suggestion
        labels: agendaLabel
        per_page: 100,
      })
```

the default page size is 30, which is probably plenty, but we should set it to the max (100). and also on the fetch for issues